torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1566,20 +1566,28 @@ Example::
   module.def(
       "_verify_model_across_ranks",
```

Comment:
curious, is it possible to consolidate `_verify_model_across_ranks` with `_verify_params_across_processes`?

Reply:
Yeah, seems that we can just delete this as we've renamed it?

Reply:
Done